locales/messages.json

```diff
@@ -272,11 +272,17 @@
   "CmdAddVersionOptSkipVersionFormatChk": "Skips the version format check.",
   "CmdAddVersionOptVerbose": "Print success messages instead of just errors.",
   "CmdContactOptSurvey": "Launch default browser to the current vcpkg survey",
-  "CmdDependInfoOptDGML": "Creates graph on basis of dgml",
-  "CmdDependInfoOptDepth": "Show recursion depth in output",
-  "CmdDependInfoOptDot": "Creates graph on basis of dot",
-  "CmdDependInfoOptMaxRecurse": "Set max recursion depth, a value of -1 indicates no limit",
-  "CmdDependInfoOptSort": "Set sort order for the list of dependencies, accepted values are: lexicographical, topological (default), x-tree, reverse",
+  "CmdDependInfoFormatConflict": "Conflicting formats specified. Only one of --format, --dgml, or --dot are accepted.",
+  "CmdDependInfoFormatHelp": "Choose output format, one of `list`, `tree`, `dot`, or `dgml`.",
+  "_CmdDependInfoFormatHelp.comment": "The alternatives in ``s must not be localized.",
+  "CmdDependInfoFormatInvalid": "--format={value} is not a recognized format. --format must be one of `list`, `tree`, `dot`, or `dgml`.",
+  "_CmdDependInfoFormatInvalid.comment": "The alternatives in ``s must not be localized. {value} is what the user specified.",
+  "CmdDependInfoOptDepth": "Show recursion depth in `list` output.",
+  "CmdDependInfoOptMaxRecurse": "Set max recursion depth. Default is no limit.",
+  "CmdDependInfoOptSort": "Choose sort order for the `list` format, one of `lexicographical`, `topological` (default), `reverse`",
+  "_CmdDependInfoOptSort.comment": "The alternatives in ``s must not be localized, but the localized text can explain what each value means. The value `reverse` means 'reverse-topological'.",
+  "CmdDependInfoShowDepthFormatMismatch": "--show-depth can only be used with `list` and `tree` formats.",
+  "CmdDependInfoXtreeTree": "--sort=xtree can not be used with formats other than tree",
```

Comment:
Same as other comment. Shouldn't this be `--sort=x-tree`

Reply:
Yep, fixed!